src/XamlStyler.dotUltimate/src/dotnet/ReSharperPlugin.XamlStyler.dotUltimate/Rider/XamlStylerModel.Generated.cs

```diff
@@ -0,0 +1,306 @@
+//------------------------------------------------------------------------------
+// <auto-generated>
```

Comment:
Is this generated on build? If so, should exclude and protect against in gitignore.

Reply:
Generated by running the `Generate Protocol` action in Rider. It's the "glue" between the IntelliJ and ReSharper world. Generation happens in Kotlin and in this C# file based on the model in `XamlStylerModel.kt`.

Reply:
Seems like maybe the article is down? I was able to access it a couple weeks back. I suppose it is not clear to me how this is different from some other generated build output file. If the file is _not_ re-generated automatically when compiling, and has to be explicitly updated, then it makes sense to keep.

Reply:
Looks like the article is down indeed. Anyway: file is generated manually, at dev time.  I'll ping CODE magazine to see what's up with that article (or publish it elsewhere)